src/lib/y2network/dialogs/edit_interface.rb

```diff
@@ -47,10 +48,13 @@ def contents
         VBox(CWM::Tabs.new(*tabs))
       end
 
+      # abort is just cancel as this is a sub dialog
       def abort_button
         Yast::Label.CancelButton
       end
 
+      # removes back button when editing device, but keep it when this dialog is
+      # followed by add followed add
```

Comment:
to many followed ;-)

May be I'd change it to something like "keep it when this dialog follows adding new interface"